.travis.yml

```diff
@@ -5,28 +5,32 @@ matrix:
   - os: linux
     env: PYTHON=2.7
   - os: linux
-    env: PYTHON=3.5
```

Comment:
Do we still need Travis?

Reply:
I don't think it hurts for now, but we should likely slim down and modernize the Travis build a bit eventually. Not my focus for now.